connect.go

```diff
@@ -196,7 +196,7 @@ func (r *Response[_]) Trailer() http.Header {
 }
 
 // internalOnly implements AnyResponse.
-func (e *Response[_]) internalOnly() {}
+func (r *Response[_]) internalOnly() {}
```

Comment:
Changed to use consistent receiver names.

Reply:
:+1: thanks! This type was briefly called "Envelope," and I must have missed this receiver name when changing.